977_squares_of_sorted_array.py

```diff
@@ -0,0 +1,9 @@
+# time: O(1)
+# space: O(1)
+from typing import List
+
+
+class Solution:
+    
+    def sortedSquares(self, nums: List[int]) -> List[int]:
+        return sorted([x**2 for x in nums])
```

Comment:
This solution is the same as your former solution, which was seeing each element and doubling it, then sorting the array. As I said before, it is inefficient and you can solve it more efficiently by using two pointers. Try the solution.